src/filters/index.ts

```diff
@@ -0,0 +1,20 @@
+export * from './base_filter.class';
+export * from './blendcolor_filter.class';
+export * from './blendimage_filter.class';
+export * from './blur_filter.class';
+export * from './brightness_filter.class';
+export * from './colormatrix_filter.class';
+export * from './composed_filter.class';
+export * from './contrast_filter.class';
+export * from './convolute_filter.class';
+export * from './filter_generator';
+export * from './gamma_filter.class';
+export * from './grayscale_filter.class';
+export * from './hue_rotation.class';
+export * from './invert_filter.class';
+export * from './noise_filter.class';
+export * from './pixelate_filter.class';
+export * from './removecolor_filter.class';
+export * from './resize_filter.class';
+export * from './saturate_filter.class';
+export * from './vibrance_filter.class';
```

Comment:
in general i would like to avoid barrel imports.
The reason for me is that they don't give me anything and they mask to me where the file is and what name it has.
For now it does not matter since this is still needeed for the bundle i suppose